src/widgets/RRule.tsx

```diff
@@ -176,7 +176,7 @@ export default function RRule(props: PropsType) {
           <FormControl>
             <InputLabel>Weekdays</InputLabel>
             <Select
-              value={sanitizeByDay(options.byday)}
+              value={sanitizeByDay(options.byday) ?? []}
```

Comment:
Please move it into sanitizeByDay which is in charge of sanitizing exactly these kind of things.